src/test/java/org/junitpioneer/jupiter/issue/IssueExtensionIntegrationTests.java

```diff
@@ -0,0 +1,91 @@
+/*
+ * Copyright 2015-2020 the original author or authors.
+ *
+ * All rights reserved. This program and the accompanying materials are
+ * made available under the terms of the Eclipse Public License v2.0 which
+ * accompanies this distribution and is available at
+ *
+ * http://www.eclipse.org/legal/epl-v20.html
+ */
+
+package org.junitpioneer.jupiter.issue;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.fail;
+
+import java.util.List;
+
+import org.junit.jupiter.api.Disabled;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.TestReporter;
+import org.junit.platform.engine.discovery.DiscoverySelectors;
+import org.junit.platform.launcher.core.LauncherDiscoveryRequestBuilder;
+import org.junit.platform.launcher.core.LauncherFactory;
+import org.junitpioneer.jupiter.Issue;
+import org.junitpioneer.jupiter.IssueTestSuite;
+import org.opentest4j.TestAbortedException;
+
+/**
+ * Mary Elizabeth Fyre: Do Not Stand at My Grave and Weep is in the public domain.
+ */
+public class IssueExtensionIntegrationTests {
+
+	private final StoringIssueProcessor issueProcessor = (StoringIssueProcessor) IssueProcessorFactory
+			.getIssueProcessors()
+			.iterator()
+			.next();
+
+	@Test
+	void testIssueCases() {
+		LauncherFactory
+				.create()
```

Comment:
What does this?

Reply:
Creates a [`Launcher`](https://junit.org/junit5/docs/5.0.3/api/org/junit/platform/launcher/Launcher.html), which gets provided with a [`LauncherDiscoveryRequest`](https://junit.org/junit5/docs/5.0.3/api/org/junit/platform/launcher/LauncherDiscoveryRequest.html) and executes it.

Essentially indirectly executes the `IssueIntegrationCases` class. By default, the `LauncherFactory` will use `ServiceLoader` to load in `TestExecutionListener`s (so `IssueExtensionExecutionListener` is automatically loaded in during the execution).